app/javascript/flavours/glitch/features/featured/index.jsx

```diff
@@ -26,145 +27,112 @@ const messages = defineMessages({
+  const acct = useAppSelector((state) => state.getIn(['accounts', accountId, 'acct']));
+  const remoteUrl = useAppSelector((state) => state.getIn(['accounts', accountId, 'url']));
+  const isAccount = useAppSelector((state) => !!(state.getIn(['accounts', accountId])));
+  const featuredItems = useAppSelector((state) => state.getIn(['user_lists', showTags ? 'featured_tags' : 'featured_accounts', accountId, 'items'], ImmutableList()));
+  const isLoading = useAppSelector((state) => !accountId || state.getIn(['user_lists', showTags ? 'featured_tags' : 'featured_accounts', accountId, 'isLoading'], true));
+  const hasMore = useAppSelector((state) => !accountId || state.getIn(['user_lists', showTags ? 'featured_tags' : 'featured_accounts', accountId, 'next'], false));
+  const suspended = useAppSelector((state) => state.getIn(['accounts', accountId, 'suspended'], false));
+  const hidden = useAppSelector((state) => getAccountHidden(state, accountId));
+
+  const handleHeaderClick = useCallback(() => columnRef.current?.scrollTop(), []);
+
+  const handleLoadMore = useCallback(() => {
+    if (showTags) dispatch(expandPinnedAccounts(accountId));
+  }, [dispatch, showTags, accountId]);
+
+  useEffect(() => {
+    if (!accountId) {
+      dispatch(lookupAccount(params.acct));
     } else {
-      dispatch(lookupAccount(acct));
+      if (!isAccount) dispatch(fetchAccount(accountId));
+      dispatch(fetchFeaturedTags(accountId));
+      dispatch(fetchPinnedAccounts(accountId, 10));
     }
-  }
 
-  componentDidUpdate (prevProps) {
-    const { params: { acct }, accountId, dispatch } = this.props;
+  }, [dispatch, showTags, isAccount, accountId, params]);
 
-    if (prevProps.accountId !== accountId && accountId) {
-      this._load();
-    } else if (prevProps.params.acct !== acct) {
-      dispatch(lookupAccount(acct));
-    }
+  if (!isLoading && !isAccount) {
+    return (
+      <BundleColumnError multiColumn={multiColumn} errorType='routing' />
+    );
   }
 
-  setRef = c => {
-    this.column = c;
-  };
-
-  handleHeaderClick = () => {
-    this.column.scrollTop();
-  };
-
-  render () {
-    const { multiColumn, intl, featuredTags, accountId, acct, isAccount, suspended, hidden, isLoading, remoteUrl } = this.props;
-
-    if (!isAccount) {
-      return (
-        <BundleColumnError multiColumn={multiColumn} errorType='routing' />
-      );
-    }
+  if (!featuredItems) {
+    return (
+      <Column>
+        <LoadingIndicator />
+      </Column>
+    );
+  }
 
-    if (!featuredTags) {
-      return (
-        <Column>
-          <LoadingIndicator />
-        </Column>
-      );
-    }
+  let emptyMessage;
 
-    /*NOTE: Limited account hint will never show, most likely because featured tags are still returned for limited accounts,
+  /*NOTE: Limited account hint will never show, most likely because featured tags are still returned for limited accounts,
```

Comment:
Nevermind, that seems to have been fixed

Reply:
Ah, no, it shows correctly when the limited user doesn't feature any tags, but when they do the tags are shown instead of the hint on reload.

Reply:
Not how that works with suspended accounts